canonical-web-and-design/ubuntu.com
# Typos and style in webinars paragraph

## Done

- “a webinars” > “a webinar”
- “third party provider” > “third-party provider”
- “third party website” > “third-party website”
- “may collects information” > “may collect information”

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/data-privacy
- Read the “Webinars” section

## Issue / Card

Fixes #6060.
